CHANGES.rst

```diff
@@ -7,6 +7,7 @@ Improvements:
  * MXKEventFormatter: E2E, hide the algo used when turning on encryption (vector-im/riot-ios#2939).
  * MXKRoomBubbleComponent: Add a property to indicate if an encryption badge should be shown.
  * MXKRoomBubbleCellData: Add a property to indicate if a bubble component needs to show encryption badge.
+ * Push notifications: Implement logic to use a secondary appId for debug builds.
```

Comment:
We can precise VoIP push as already have it for APNS :slightly_smiling_face:

Reply:
Updated.